server/util/db/db.go

```diff
@@ -193,7 +197,17 @@ func openDB(configurator *config.Configurator, dialect string, connString string
 	}
 
 	var l logger.Interface
-	l = sqlLogger{Interface: logger.Default, logLevel: logger.Warn}
+	// This is the same as logger.Default, but with colors turned off (since the
+	// output may not support colors) and sending output to stderr (to be
+	// consistent with the rest of our logs).
+	gormLogger := logger.New(
+		golog.New(os.Stderr, "\r\n", golog.LstdFlags),
```

Comment:
this is an improvement for sure! I wonder if there's a way to get a "Writer" from our server/util/log package and use that, so everything flows through the configured zerolog logger?

Reply:
Done -- for simplicity, logging everything at info level for now (instead of allowing it to be configurable).

Reply:
Actually I think with the current code it will log everything at INFO level, which isn't desirable for the "SLOW QUERY" warnings. Let me see if I can fix that...

Reply:
The fix looks like it will be somewhat involved -- reverted everything back to golog for now and we can address in a followup PR.

Reply:
FWIW the existing GORM logs don't have an "actual" severity level but rather print it as part of the log message so logging it all at our INFO severity wouldn't be a change in practice. In any case I would suggest leaving other changes, if any, to a separate PR.

Reply:
:+1: